enigma-coinjoin-client/src/index.js

```diff
@@ -4,13 +4,12 @@ const {PUB_KEY_UPDATE, SUBMIT_DEPOSIT_METADATA, SUBMIT_DEPOSIT_METADATA_SUCCESS,
 const EventEmitter = require('events');
 const Web3 = require('web3');
 const {utils} = require('enigma-js/node'); // TODO: Replace by browser version before bundling
-const WebSocket = require('ws');
```

Comment:
IFAIK - The `WebSocket` dependency is required to test with mocha.
```
  1) Contract: Mixer
       "before all" hook:
     ReferenceError: WebSocket is not defined
      at new CoinjoinClient (enigma-coinjoin-client/src/index.js:14:23)
      at Context.before (test/api.test.js:28:15)
```
As indicated in the README of the client module, I suggest that we continue assuming node for the sources of this module. But we should bundle it for the browser (using Broserify or WebPack).

Consider these two hot module replacements when bundling (in the bundle, not the source):
* Remove the `WebSocket` dependency
* Replace `enigma-js/node` by `enigma-js/browser`

Let me know if you have other suggestions but I believe that this is the simplest approach.

Reply:
Fixed the scope of variable declaration, it seems to work now.